src/persistence.js

```diff
@@ -20,6 +20,9 @@ export default function _Persistence(mpInstance) {
     };
 
     this.initializeStorage = function() {
+        if (mpInstance._Store.getPrivacyFlag('SDKState')) {
```

Comment:
I think we still would want to know that `mpInstance._Store.isFirstRun = true` based on if the customer had been here in the past prior to setting the nolocation and notargeting.  We don't actively delete the cookies, (and while this information is likely not valuable). We'd send a boolean for isFirstRun to the backend rather than null which is what is happening now.

<img width="963" height="259" alt="Image" src="https://github.com/user-attachments/assets/0e37f688-52e4-4ced-bb31-836571612d7f" />